rerun_cpp/src/archetypes/fuzzy.hpp

```diff
@@ -0,0 +1,9 @@
+// NOTE: This file was autogenerated by re_types_builder; DO NOT EDIT.
+// Based on "definitions/rerun/testing/archetypes/fuzzy.fbs"
```

Comment:
I thin we should add this line to Rust and Python generated code as well, but their source was a bit too obscure for me to attempt it